src/Frontend/Page/Admin.hs

```diff
@@ -292,16 +292,25 @@ instance ToHtml PageAdminSettingsGaPUsersView where
                             input_ [type_ "text", class_ "inline-search-input", value_ "Nutzersuche"] -- FIXME Placeholder not value
                             a_ [href_ U.Broken, class_ "inline-search-button"] $ i_ [class_ "icon-search"] nil -- FIXME dummy
 
-                let renderUserRow :: forall m. (Monad m) => User -> HtmlT m ()
-                    renderUserRow user = tr_ $ do
-                        td_ . span_ [class_ "img-container"] $ avatarImgFromMaybeURL (user ^. userAvatar)
+                let renderUserInfoRow :: forall m. (Monad m) => User -> HtmlT m ()
+                    renderUserInfoRow user = do
                         td_ $ user ^. userLogin . unUserLogin . html
                         td_ (case user ^. userRole of
                                 Student cl    -> toHtml $ showSchoolClass cl
                                 ClassGuest cl -> toHtml $ showSchoolClass cl
                                 _             -> nil)
                         td_ $ roleLabel (user ^. userRole)
                         td_ (toHtmlRaw nbsp)
+
+                let renderUserRow :: forall m. (Monad m) => UserView -> HtmlT m ()
+                    renderUserRow (DeletedUser user) = tr_ $ do
+                        td_ nil
```

Comment:
Why not show the avatar of deleted users?  (Just curious, we can keep it this way.)